### PR TITLE
chore: bumped actions/labeler to v5

### PR DESCRIPTION
**Pull request by kvanzuijlen**
## What
Bumped actions/labeler to v5

## Why
v5 is now GA

## References
https://github.com/actions/labeler

### CODE REVIEW

File: .github/labeler.yaml [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-- 'LICENSE'
+  - changed-files:
+      - any-glob-to-any-file:
+        - 'LICENSE'
 
 tests:
-- '**/*_test.go'
+  - changed-files:
+      - any-glob-to-any-file:
+        - '**/*_test.go'
```
Comment (bschaatsbergen) [inline .github/labeler.yaml:26]: If it's "any glob to any file" isn't this annotation then incorrect?

Comment (kvanzuijlen) [reply .github/labeler.yaml:26]: According to the docs it works as follows: any-glob-to-any-file: ANY glob must match against ANY changed file

Comment (bschaatsbergen) [reply .github/labeler.yaml:26]: I'm just going ahead and approve this. It's super unclear, even after reading the docs myself I still find it overly vague.

**Review by bschaatsbergen — APPROVED** [review]
LGTM 🚀

**Comment by bschaatsbergen** [comment]
Thanks for your contribution @kvanzuijlen 👏

File: .github/labeler.yaml (hunk 1 above)
Comment (kvanzuijlen) [reply .github/labeler.yaml:26]: 💯

**Pull request merged**